spec/Main.hs

```diff
@@ -484,12 +487,14 @@ liveTests =
        it "should be able to query custom taxonomies" $ do
          ("department", ctxt) `shouldRenderContaining` "A sports post"
          ("department", ctxt) `shouldNotRenderContaining` "A first post"
-       it "should be able to query custom endpoints" $ do
+       it "should be able to query custom endpoints (as object)" $ do
          ("custom-endpoint-object", ctxt) `shouldRenderContaining` "categories"
          ("custom-endpoint-object", ctxt) `shouldNotRenderContaining` "departments"
-       it "should be able to query custom endpoints" $ do
+       it "should be able to query custom endpoints (as array)" $ do
          ("custom-endpoint-array", ctxt) `shouldRenderContaining` "2014-10-01"
          ("custom-endpoint-array", ctxt) `shouldRenderContaining` "2014-10-02"
          ("custom-endpoint-array", ctxt) `shouldRenderContaining` "2014-10-15"
        it "should be able to reference fields from the custom endpoint in another custom endpoint query" $ do
          ("custom-endpoint-array", ctxt) `rendersSameAs` "custom-endpoint-enter-the-matrix"
+       it "should handle 404s without blowing up" $
+         ("custom-endpoint-404", ctxt) `shouldRenderContaining` "404"
```

Comment:
This works, but the wreq/http-client exception has a LOT of information that doesn't belong in the HTML debugging comment. Right now all the information about the request is in there.